src/sortStudents.ts

```diff
@@ -1,16 +1,53 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age ='age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function averageMarks(marks: number[]): number {
+  return marks
+    .reduce((total: number, current: number): number => total + current, 0)
```

Comment:
You don't need this types, array already typified above ⬆️ 
```suggestion
    .reduce((acc ,current): number => acc + current, 0)
```